package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Some experiments after seeing Rust PR #45905.",
   "main": "do-the-wasm.js",
   "scripts": {
-    "prestart": "make",
+    "rust-sample": "make --directory=examples/rust",
```

Comment:
Can you run this on CI and make sure the sample tests are executed?

Reply:
this is not a test any more, it is an example. we can run this step on the CI... Do you want to have this as a test ?

Reply:
I want to have every single line of code in this repo tested on CI :) Which includes `examples/rust/test.js`

Reply:
can't agree more... 👍